Don't use '-' to make make exit on error in subdir makes.

Index: Makefile
===================================================================
--- Makefile.orig	2009-11-26 18:36:55.000000000 +0100
+++ Makefile	2009-11-26 18:49:09.674873998 +0100
@@ -31,11 +31,11 @@ DIRS = irattach irdaping etc man irnetd
 CFLAGS= -O2 -W -Wall
 
 all:
-	@-(set -e ; for d in $(DIRS) ; do $(MAKE) $(MAKE_OUTPUT) -C $$d $@ ; done)
+	@(set -e ; for d in $(DIRS) ; do $(MAKE) $(MAKE_OUTPUT) -C $$d $@ ; done)
 
 
 install:
-	@-(set -e ; for d in $(DIRS) ; do $(MAKE) $(MAKE_OUTPUT) -C $$d $@ ; done)
+	@(set -e ; for d in $(DIRS) ; do $(MAKE) $(MAKE_OUTPUT) -C $$d $@ ; done)
 
 
 clean:
